cms/log.py

```diff
@@ -40,8 +40,8 @@
 # OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 #
 # You can find the original files at:
-# http://hg.python.org/cpython/file/69ee9b554eca/Lib/logging/__init__.py
-# http://hg.python.org/cpython/file/69ee9b554eca/Lib/logging/handlers.py
+# https://hg.python.org/cpython/file/f23fa1f7b68f/Lib/logging/__init__.py
+# https://hg.python.org/cpython/file/4243df51fe43/Lib/logging/handlers.py
```

Comment:
Minor note: The new links are for two different revisions. Is it intended?

Reply:
Maybe - I think I clicked on one I clicked on the "last commit changing this file" to get a permalink instead of the "last commit", I'll change it.